### ISSUE TITLE
Few-shot VideoQA training details

### ISSUE THREAD

**Issue by engindeniz** [antoyang/FrozenBiLM#9]
Hi,

Thanks for the great work and publicly available code.

Could you please share the few-shot training parameters (batch size, learning rate, etc.)? I could not reproduce the results. 

Thanks in advance.

**Comment by antoyang** [comment]
Hi,
Thanks for your interest! For the few-shot setting, I ran the same hyperparameter swipes as for the fully-supervised setting (see Section 4.1 and Appendix Section C.2).
